### PR TITLE
Update README and versions for 20.06 release

### PR DESCRIPTION
**Pull request by dzier**
Don't merge until release

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -30,13 +30,256 @@
...
+inferencing for any model being managed by the server. For edge
+deployments, Triton Server is also available as a shared library with
+an API that allows the full functionality of the server to be included
+directly in an application. 
+
+What's New in 2.0.0
+-------------------
+
+* Updates for KFserving HTTP/REST and GRPC protocols and corresponding Python 
+  and C++ client libraries. This release includes support for both the new
```
Comment (deadeyegoodwin) [inline README.rst:57]: Delete from  "This release includes..."

And delete the following 2 bullets.

File: README.rst [hunk 2]
```diff
@@ -30,13 +30,256 @@
...
+  libraries is released on GitHub branch r20.06 and as NGC container 20.06-py3.
+
+* Support for the legacy V1 HTTP/REST, GRPC and corresponding client libraries 
+  is released on GitHub branch r20.06-v1 and as NGC container 20.06-v1-py3.
+
+* Migration from Triton V1 to Triton V2 requires signficant changes, see the 
+  “Backwards Compatibility” and “Roadmap” sections of the GitHub README for more
+  information.
+
+* Refer to the 20.06 column of the  Frameworks Support Matrix for container
```
Comment (deadeyegoodwin) [inline README.rst:70]: Delete this bullet as well.

File: README.rst [hunk 3]
```diff
@@ -30,13 +30,256 @@
...
+The current release of the Triton Inference Server is 2.0.0 and
+corresponds to the 20.06 release of the tensorrtserver container on
+`NVIDIA GPU Cloud (NGC) <https://ngc.nvidia.com>`_. The branch for
+this release is `r20.06
+<https://github.com/NVIDIA/triton-inference-server/tree/r20.06>`_.
+
+Backwards Compatibility
+-----------------------
+
+Version 2 of Triton does not generally maintain backwards
```
Comment (deadeyegoodwin) [inline README.rst:154]: This has been updated on master branch. You should base this README off the latest master README

File: README.rst [hunk 4]
```diff
@@ -30,13 +30,256 @@
...
+In the 20.06 V2 release there will be some changes to the tritonserver
+command-line executable arguments to remove deprecated arguments and
+adjust defaults. The changes will be detailed as part of the 20.06
+release. It will be necessary to revisit and possibly adjust
+invocations of tritonserver executable.
+
+Documentation
+-------------
+
+The documentation is in the process of being updated for version 2. As
```
Comment (deadeyegoodwin) [inline README.rst:206]: This is updated on master

File: README.rst (hunk 3 above)
Comment (dzier) [reply README.rst:154]: Is that valid for 20.06? I am using the README that we in master just prior to the 20.06 freeze.

File: README.rst [hunk 5]
```diff
@@ -30,13 +30,286 @@
...
+The 20.03.1 release of Triton consists of a single server/container
+that supports both the existing version 1 APIs and protocols and the
+new version 2 APIs and protocols. For version 2 the release is beta
+quality and includes the new `HTTP/REST and GRPC protocols
+<https://github.com/kubeflow/kfserving/tree/master/docs/predict-api/v2>`_
+and corresponding new C++ and Python client libraries. Version 2 also
+includes a beta release of the new server C API defined in
+tritionserver.h.
+
+The upcoming 20.06 release of Triton will include two separate server
```
Comment (deadeyegoodwin) [inline README.rst:205]: Need present tense since this is the release.
"The 20.06 release of Triton includes ..."

File: README.rst [hunk 6]
```diff
@@ -30,13 +30,286 @@
...
+containers:
+
+* A legacy V1 version of Triton will be released from the master-v1
```
Comment (deadeyegoodwin) [inline README.rst:208]: "A legacy V1 version of Triton is released from the r20.06-v1 branch."

"... version of Triton is called ..."

File: README.rst [hunk 7]
```diff
@@ -30,13 +30,286 @@
...
+  branch. The NGC container for the V1 version of Triton will be
+  called tritonserver:20.06-v1-py3. The V1 version of Triton is
+  deprecated and no releases beyond 20.06 are planned. The V1 version
+  of Triton maintains backwards compatibility with prior V1 versions
+  in both the server APIs and in the C++ and Python libraries. See the
+  `master-v1 branch README
+  <https://github.com/NVIDIA/triton-inference-server/tree/master-v1>`_
+  for more information.
+
+* The new V2 version of Triton will be released from the master branch
```
Comment (deadeyegoodwin) [inline README.rst:218]: "... is released from the r20.06 branch and includes the..."

File: README.rst [hunk 8]
```diff
@@ -30,13 +30,286 @@
...
+  and will include the new GRPC and HTTP protocols based on `inference
+  protocols
+  <https://github.com/kubeflow/kfserving/tree/master/docs/predict-api/v2>`_
+  that have been proposed by the `KFServing project
+  <https://github.com/kubeflow/kfserving>`_. Version 2 of Triton will
```
Comment (deadeyegoodwin) [inline README.rst:223]: "... Triton has a new C API and new ...."

".. NGC container... is called. ..."

File: README.rst [hunk 9]
```diff
@@ -30,13 +30,286 @@
...
+  also have a new C API and new C++ and Python client libraries. The
+  NGC container for version 2 of Triton will be called
+  tritonserver:20.06-py3.
+
+For both V1 and V2 the model repository struture and custom backend
+APIs will remain unchanged so that any existing model repository and
+custom backends will continue to work with Triton Server.
+
+In the 20.06 V2 release there will be some changes to the tritonserver
```
Comment (deadeyegoodwin) [inline README.rst:232]: Delete this paragraph

**Pull request merged**